.github/pull_request_template.md

```diff
@@ -2,6 +2,12 @@
 
 ## Which issue(s) this PR fixes
```

Comment:
maybe here also should be _...PR closes_ ? I think "Closes" is more generic word that covers both bug fixes and improvements, wdyt?

Reply:
good point 👍